### PR TITLE
Btn profile name, register, user profile bug fixes from dev branch

### PR DESCRIPTION
**Pull request by revathip**

### CODE REVIEW

File: library/ws-widget/collection/src/lib/btn-profile/btn-profile.component.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 <button mat-button i18n-matTooltip matTooltip="User Menu" class="new-margin" #rootTrigger="matMenuTrigger"
   [matMenuTriggerFor]="settingsMenu">
   <mat-icon>account_circle</mat-icon>
-  <span class="padding-left-xs" i18n>{{ givenName || ''}}</span>
+  <span class="padding-left-xs" i18n>{{ name || 'Guest'}} </span>
```
[the remark this answers is not in the document]
Comment (aastarurmika) [reply library/ws-widget/collection/src/lib/btn-profile/btn-profile.component.html:16]: @revathip

File: src/app/routes/forgot-password/forgot-password.component.ts [hunk 2]
```diff
@@ -68,10 +87,13 @@ export class ForgotPasswordComponent implements OnInit {
           }
         },
         (error: any) => {
-          this.openSnackbar(error.error.error)
+          if (error.error.error === 'User Not Found') {
+            this.openSnackbar('User data doesnot exist')
```
[the remark this answers is not in the document]
Comment (aastarurmika) [reply src/app/routes/forgot-password/forgot-password.component.ts:91]: @revathip

File: src/app/routes/forgot-password/forgot-password.component.ts [hunk 3]
```diff
@@ -68,10 +87,13 @@ export class ForgotPasswordComponent implements OnInit {
...
+          } else {
+            this.openSnackbar(error.error.error)
```
[the remark this answers is not in the document]
Comment (aastarurmika) [reply src/app/routes/forgot-password/forgot-password.component.ts:93]: @revathip

**Review by aastarurmika — CHANGES_REQUESTED** [review]

**Comment by revathip** [comment]
@ankitv89 , review comments are completed.

**Comment by ankitv89** [comment]
@revathip do we still need to merge this PR or can we close this?

**Comment by revathip** [comment]
Feature is already pushed and deployed

**Pull request closed without merging**